nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceProvider.java

```diff
@@ -70,7 +67,7 @@
 
     private final ProcessScheduler processScheduler;
     private static final Set<Method> validDisabledMethods;
-    private final BulletinRepository bulletinRepo;
+    // private final BulletinRepository bulletinRepo;
```

Comment:
I assume so, this change is du to changes from Mark and Matt this morning. I probably should remove comment